spec/beautify-languages-spec.coffee

```diff
@@ -26,7 +26,7 @@ describe "BeautifyLanguages", ->
     "mustache", "objective-c", "perl", "php",
     "python", "ruby", "sass", "sql", "svg",
     "xml", "csharp", "gfm", "marko",
-    "go", "html-swig"
+    "go", "html-swig", "lua"
```

Comment:
You will need to install `language-lua` package as well. See https://github.com/Glavin001/atom-beautify/blob/master/.travis.yml#L25